pkg/dbal/Tests/Client/DbalDriverTest.php

```diff
@@ -118,11 +119,29 @@ public function testShouldConvertTransportMessageToClientMessage()
         $this->assertSame('ContentType', $clientMessage->getContentType());
         $this->assertSame(1000, $clientMessage->getTimestamp());
         $this->assertSame(12345, $clientMessage->getDelay());
-
-        $this->assertNull($clientMessage->getExpire());
+        $this->assertSame(67890, $clientMessage->getExpire());
         $this->assertSame(MessagePriority::NORMAL, $clientMessage->getPriority());
     }
 
+    public function testShouldRoundTransportMessageTimeToLiveToNearestIntegerWhenConvertingToClientMessage()
+    {
+        $driver = new DbalDriver(
+            $this->createPsrContextMock(),
+            $this->createDummyConfig(),
+            $this->createDummyQueueMetaRegistry()
+        );
+
+        $transportMessage = new DbalMessage();
+
+        $transportMessage->setTimeToLive(1.4);
+        $clientMessage = $driver->createClientMessage($transportMessage);
+        $this->assertSame(1, $clientMessage->getExpire());
+
+        $transportMessage->setTimeToLive(1.5);
```

Comment:
FYI. It is 1.5 milliseconds

Reply:
I did this because the $transportMessage value can be `null|int|float`, but $clientMessage can only be `null|int`. Despite $clientMessage claiming to be in seconds, the current behavior of the DbalDriver is to not perform the conversion to milliseconds.